### PR TITLE
Single place to get environment info

### PR DESCRIPTION
**Pull request by gabrielfs7**
Single place to get environment info.

https://oat-sa.atlassian.net/browse/NEX-1095

### CODE REVIEW

File: models/classes/webhooks/WebhookEventsService.php [hunk 1]
```diff
@@ -94,6 +95,8 @@ public function handleEvent(Event $event)
             return;
         }
 
+        $this->logEvent('Handle', $event); //@TODO FIXME Remove after test
```
Comment (mike-ionut-mihai-sandu-tao) [inline models/classes/webhooks/WebhookEventsService.php:98]: ?

Comment (gabrielfs7) [reply models/classes/webhooks/WebhookEventsService.php:98]: Will be removed after tests :)

Comment (gabrielfs7) [reply models/classes/webhooks/WebhookEventsService.php]: Removed

**Comment by gabrielfs7** [comment]
> we have some leftovers for testing , but the code looks like what we agreed on , so it is ok

@mike-ionut-mihai-sandu-tao I added test where was possible. Thanks for the review!

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: models/classes/webhooks/configEntity/Webhook.php [hunk 2]
```diff
@@ -57,22 +58,26 @@ class Webhook implements WebhookInterface
...
+        array $extraPayload = []
+    )
     {
         $this->id = $id;
         $this->url = $url;
         $this->httpMethod = $httpMethod;
         $this->auth = $auth;
         $this->retryMax = $retryMax;
         $this->responseValidation = $responseValidation;
+        $this->extraPayload = $extraPayload;
```
Comment (siwane) [inline models/classes/webhooks/configEntity/Webhook.php:80]: dynamic declaration must be avoided

**Comment by bartlomiejmarszal** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**